### PR TITLE
Updates to flat data and zero copy example

### PR DESCRIPTION
**Pull request by harishkumaruk**

### CODE REVIEW

File: examples/connext_dds/flat_data_latency/c++11/CameraImage_publisher.cxx [hunk 1]
```diff
@@ -119,8 +119,12 @@ void publisher_flat(const ApplicationOptions &options)
             }
         }
     }
-    std::cout << "Average end-to-end latency: "
+    if (count) {
```
Comment (alexcamposruiz) [inline examples/connext_dds/flat_data_latency/c++11/CameraImage_publisher.cxx]: `count > 0` for clarity.

Comment (alexcamposruiz) [reply examples/connext_dds/flat_data_latency/c++11/CameraImage_publisher.cxx]: should we move it to a function? print_latency(total_latency, count) ?

Comment (harishkumaruk) [reply examples/connext_dds/flat_data_latency/c++11/CameraImage_publisher.cxx]: Updated the changes

File: examples/connext_dds/flat_data_latency/c++11/Common.hpp [hunk 2]
```diff
@@ -170,3 +129,13 @@ inline void configure_nic(
                 nic});
     }
 }
+
+inline void print_latency(const int &total_latency, const int &count)
```
Comment (harishkumaruk) [inline examples/connext_dds/flat_data_latency/c++11/Common.hpp]: Added print_latency function

Comment (alexcamposruiz) [reply examples/connext_dds/flat_data_latency/c++11/Common.hpp]: Pass arguments by value (they are primitives): print_latency(int, int)

Comment (alexcamposruiz) [reply examples/connext_dds/flat_data_latency/c++11/Common.hpp]: See rules F.15 and F.16 in https://bitbucket.rti.com/projects/CONNEXT/repos/cpp_core_guidelines/browse/CppCoreGuidelines.md#

File: examples/connext_dds/flat_data_latency/c++11/Common.hpp [hunk 3]
```diff
@@ -130,7 +130,7 @@ inline void configure_nic(
     }
 }
 
-inline void print_latency(const int &total_latency, const int &count)
+inline void print_latency(int total_latency, int count)
```
Comment (harishkumaruk) [inline examples/connext_dds/flat_data_latency/c++11/Common.hpp]: Updated to pass by value

**Pull request merged**